Modelica/Blocks/Sources.mo

```diff
@@ -118,12 +118,12 @@ variable <strong>y</strong> is both a variable and a connector.
 
   end BooleanExpression;
 
-  block Clock "Generate actual time signal"
+  block ContinuousClock "Generate actual time signal"
     extends Interfaces.SignalSource;
 
   equation
     y = offset + (if time < startTime then 0 else time - startTime);
-    annotation (
+    annotation (defaultComponentName="clock",
       Icon(coordinateSystem(
```

Comment:
I thought the reason for this PR was to not use "Clock" in MSL. Now having instances being called "clock" might lead users to also use "Clock". Maybe?

Reply:
I tried to be minimal invasive and only changed the class name while keeping all instance names.

Reply:
But by adding this you encourage the use of "clock". So I would not add the defaultComponentName annotation of you don't want to be to invasive and educating people to avoid "Clock" and therefore also "clock"

Reply:
OK, will also change/avoid `clock` as instance name and update the PR later.

Reply:
I actually think it's ok with `clock` as component name.  As soon as `Clock` becomes a keyword, this is not going to be an issue any more, and then `clock` is a very natural component name.

Reply:
Unfortunately to this day, Windows users still haven't grasped that Clock is not clock. So better try to help avoid this.

Reply:
I also think that clock should be ok, even if we want to avoid Clock. 
We have the reverse issue with the MathBoolean blocks where the classes are named And, Or, Not - even though and, or, not are reserved words.

Reply:
> I also think that clock should be ok, even if we want to avoid Clock.

Yes, that is what I considered first. Now, I am in limbo how to proceed? Leave at is (as @HansOlsson approved) or rename `clock` and undo the defaultComponentName (as requested by @dietmarw and @henrikt-ma). We have draw here - two votes for either option.

Reply:
Maybe renaming existing models goes a bit far but I would still argue for not introducing the "defaultComponentName".

Reply:
I didn't want to rename `clock`.  I wanted to rename components called `Clock` to `clock`.

Reply:
> Maybe renaming existing models goes a bit far but I would still argue for not introducing the "defaultComponentName".

OK.

> I didn't want to rename `clock`. I wanted to rename components called `Clock` to `clock`.

#2945 only was on the class name. That is what I want to fix here.

Reply:
OK, so I suggested a scope creep of #2945.  Without doing all of it, what's the point of making any change at all?

Reply:
To be clear: as I understand there are two examples (Modelica.Electrical.Digital.Examples.Counter3 and Modelica.Electrical.Digital.Examples.Counter) that use Clock as a component-name,

Those examples break naming conventions (components should start with lower-case letter) and I think it makes sense to have a larger PR for fixing all of them (really soon now). Whether we fix Clock in this PR - or do it together with all other names shouldn't be such a big issue, and I see a benefit of fixing all component names in those examples in one PR.

Reply:
Sure, we have #283 for the naming conventions (Yes, more than 9 years old.)

Reply:
And could merge this PR already now.